### PR TITLE
gradle: Add spanish translation

### PR DESCRIPTION
**Pull request by vfrico**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Hello, I've added the translation in spanish for gralde page.

A note for spanish reviewers: Which is the rule that applies for spanish translation in verbs? I've readed some translations and some uses the "indicative" form and others "present simple". Some people comments in other pull requests and suggests to use the spanish present simple in every translation, but I could not found any rule

### CODE REVIEW

File: pages.es/common/gradle.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# gradle
+
+> Gradle es un sistema de código abierto para automatizar la compilación de proyectos
+> Más información en: <https://gradle.org>.
+
+- Compilar un proyecto:
```
Comment (navarroaxel) [inline pages.es/common/gradle.md:6]:
```suggestion
- Compila un proyecto:
```

File: pages.es/common/gradle.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`gradle build`
+
+- Excluit la tarea *test*:
```
Comment (navarroaxel) [inline pages.es/common/gradle.md:10]:
```suggestion
- Excluye la tarea `test`:
```

File: pages.es/common/gradle.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`gradle build -x {{test}}`
+
+- Ejecutar en modo offline para prevenir que gradle acceda a la red durante una compilación:
```
Comment (navarroaxel) [inline pages.es/common/gradle.md:14]:
```suggestion
- Ejecuta en modo offline para prevenir que gradle acceda a la red durante una compilación:
```

File: pages.es/common/gradle.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`gradle build --offline`
+
+- Limpiar el directorio de compilación:
```
Comment (navarroaxel) [inline pages.es/common/gradle.md:18]:
```suggestion
- Limpia el directorio de compilación:
```

File: pages.es/common/gradle.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`gradle clean`
+
+- Compilar y generar un paquete:
```
Comment (navarroaxel) [inline pages.es/common/gradle.md:22]:
```suggestion
- Compila y genera un paquete:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @vfrico, welcome and thanks for your submission. We use the present indicative for the example descriptions. I left some comments.

**Comment by vfrico** [comment]
I've pushed a new commit with the new changes. Thanks!

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — COMMENTED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Review by sbrl — COMMENTED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Review by sbrl — APPROVED** [review]

**Pull request merged**